x/protocolpool/keeper/msg_server.go

```diff
@@ -22,6 +22,44 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
 	return &MsgServer{Keeper: keeper}
 }
 
+func (k MsgServer) ClaimBudget(ctx context.Context, msg *types.MsgClaimBudget) (*types.MsgClaimBudgetResponse, error) {
```

Comment:
we should add in the readme that this module is lazy and users need to actively claim funds instead of it being automatic